packages/storage-postgres/prisma/migrations/20240923233444_init/migration.sql

```diff
@@ -0,0 +1,13 @@
+-- CreateTable
+CREATE TABLE "WorkflowState" (
+    "workflowId" TEXT NOT NULL,
```

Comment:
```suggestion
    "workflowId" UUID NOT NULL,
```

see other comment on db type